src/rules/enforceTrailingCommaRule.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2013 Palantir Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static FAILURE_STRING = "missing trailing comma";
+
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        return this.applyWithWalker(new EnforceTrailingCommaWalker(sourceFile, this.getOptions()));
+    }
+}
+
+class EnforceTrailingCommaWalker extends Lint.RuleWalker {
+    public visitObjectLiteralExpression(node: ts.ObjectLiteralExpression) {
+        const child = node.getChildAt(1);
+        if (child != null && child.kind === ts.SyntaxKind.SyntaxList) {
+
+            if (child.getText().match(/\n|\r/)) {
```

Comment:
I like the options ESLint uses: http://eslint.org/docs/rules/comma-dangle
"always", "always-newline", "never"

Maybe we could add options similar to those? Enforcing trailing commas for multiline literals but not single line literals might not be what everyone wants